.gitignore

```diff
@@ -1,6 +1,7 @@
 *~
 .class
 .metadata
+.settings
```

Comment:
I'd prefer to keep the .settings file here to enforce certain project settings in Eclipse